zqd/ingest/log.go

```diff
@@ -158,7 +154,8 @@ func (p *LogOp) Status() <-chan string {
 	return p.warningCh
 }
 
+// Error indicates what if any error occurred during import, after the
+// Status channel is closed.
```

Comment:
NIt: Maybe state explicitly that this returns an undefined value if the Status channel is still open?